docs/management/index.md

```diff
@@ -421,10 +421,14 @@ The management UI shows now a username/password login form for Basic Authenticat
 ![Single OAuth 2.0 resource, with oauth_disable_basic_auth = false](./management-oauth-with-basic-auth.png)
 
 
-### Logging out of the Managment UI {#about-logout-workflow}
+### Logging out of the management UI {#about-logout-workflow}
 
 RabbitMQ follows the [OpenID Connect RP-Initiated Logout 1.0](https://openid.net/specs/openid-connect-rpinitiated-1_0.html)
-specification to implement the logout workflow. This means that the logout workflow is triggered from the management UI when the user clicks on the **Logout** button. Logging out from management UI not only logs the user out from the management UI itself but also from the Identity Provider.
+specification to implement the logout workflow provided the [OpenId Connect Discovery endpoint](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationRequest) returns the attribute `end_session_endpoint` on the response. It works as follows:
+
+  1. The user clicks on the **Logout** button,
+  2. If the management UI found an `end_session_endpoint` from the [OpenId Connect Discovery endpoint](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationRequest), the management UI sends a logout request to the Idp following the [OpenID Connect RP-Initiated Logout](https://openid.net/specs/openid-connect-rpinitiated-1_0.html) specification
```

Comment:
change "found" to "finds"

Reply:
What does this mean: "following the [OpenID Connect RP-Initiated Logout](https://openid.net/specs/openid-connect-rpinitiated-1_0.html) specification"??? 

the logout request follows the guidelines in the OpenID Connect RP-Initiated Logout](https://openid.net/specs/openid-connect-rpinitiated-1_0.html) specification?? maybe clarify this a little further

Add full stop at the end of the step

Reply:
I have simplified the paragraph but it means that RabbitMQ implements the steps indicated by the specification in order to logout users.